tests/helpers.py

```diff
@@ -135,20 +138,23 @@ def create_pod(
         pod_data['metadata']['labels']['app'] = pod_name
         pod_data['spec']['template']['metadata']['labels']['name'] = pod_name
         pod_data['spec']['replicas'] = replica_count
+    if attach_multi_pvc and not raw_block_pv:
+        pod_data = attach_multiple_pvc_single_pod(pod_data, pvc_list)
+    else:
+        if pvc_name:
+            if dc_deployment:
+                pod_data['spec']['template']['spec']['volumes'][0][
+                    'persistentVolumeClaim'
+                ]['claimName'] = pvc_name
+            else:
+                pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = pvc_name
 
-    if pvc_name:
-        if dc_deployment:
-            pod_data['spec']['template']['spec']['volumes'][0][
-                'persistentVolumeClaim'
-            ]['claimName'] = pvc_name
+    if raw_block_pv:
+        if attach_multi_pvc:
+            pod_data = attach_multiple_pvc_single_pod(pod_data, pvc_list, raw_block_pv=True)
```

Comment:
This can be added in line 142.
```
if attach_multi_pvc:
    pod_data = attach_multiple_pvc_single_pod(pod_data, pvc_list, raw_block_pv=raw_block_pv)
```

Reply:
thanks, will update